src/nidmm/metadata/__init__.py

```diff
@@ -3,7 +3,7 @@
 from metadata.attributes import attributes
 from metadata.enums import enums
 
-import build.metadata_utilities
+import build.helper as helper
```

Comment:
"build" and "helper" are very generic names.
I think it's time we name our code generator and have the module use that name. For example: say we need to break up the repo. You're in nidmm-python repo... what would you need to import?

What do you think of nimigen or nimi-gen ?

Reply:
How about calling it marc? That way it's almost each of our names, but not quite...